fastai/optimizer.py

```diff
@@ -11,12 +11,24 @@
 # Cell
 class _BaseOptimizer():
     "Common functionality between `Optimizer` and `OptimWrapper`"
-    def all_params(self, n=slice(None), with_grad=False):
+    def all_params(self,
```

Comment:
Needs a docstring and type annotations

Reply:
I dont know why this shows, but I believe only the one below is on code (I will check later the PR just to see if this line is not a duplicate as there is only 1 `all_params` on `_BaseOptimizer`.

Reply:
using `(slice, int)`